### DISCUSSION TITLE
[Feature] Immich machine learning multithreading

### DISCUSSION THREAD

**Discussion by lordraiden** [immich-app/immich#4276] [Feature Request]
### The feature

I see that mostly when the jobs are running with many pics only 1 cpu core is used, it would be nice to be able to scale to several cores in parallel 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
@mertalev this should already spread across cores by default, right?

**Reply by mertalev** [reply]
It does. Each request is only given one thread, though, so you need to increase the job concurrency to use more cores.
